libtenzir/builtins/operators/files.cpp

```diff
@@ -216,12 +217,27 @@ class files_operator final : public crtp_operator<files_operator> {
   files_args args_ = {};
 };
 
-class plugin final : public virtual operator_plugin<files_operator> {
+class plugin final : public virtual operator_plugin<files_operator>,
+                     public virtual operator_factory_plugin {
 public:
   auto signature() const -> operator_signature override {
     return {.source = true};
   }
 
+  auto make(invocation inv, session ctx) const
+    -> failure_or<operator_ptr> override {
+    auto args = files_args{};
+    argument_parser2::operator_("files")
+      .add(args.path, "<path>")
+      .add("recurse", args.recurse_directories)
+      .add("follow_symlinks", args.follow_directory_symlink)
+      //XXX: Reconsider default
```

Comment:
I've thought more about our discussion yesterday. I think removing the option, skipping by default, and then warning per entry where the permission denied makes the most sense. What do you think?

Reply:
Makes sense. It could be combined with the new `--strict` flag perhaps by the user?

Reply:
Yes, I think that'd work.

Reply:
I will change this later in a follow-up, if that's fine.